diff -ruN boost-src/libs/pfr/CMakeLists.txt boost-src-modified/libs/pfr/CMakeLists.txt
--- boost-src/libs/pfr/CMakeLists.txt	2025-04-10 10:04:37.000000000 -0700
+++ boost-src-modified/libs/pfr/CMakeLists.txt	2025-06-08 15:43:36.571879942 -0700
@@ -11,7 +11,7 @@
 add_library(boost_pfr INTERFACE)
 add_library(Boost::pfr ALIAS boost_pfr)
 
-target_include_directories(boost_pfr INTERFACE include)
+target_include_directories(boost_pfr SYSTEM INTERFACE include)
 
 if (CMAKE_VERSION VERSION_GREATER_EQUAL "3.28.0" AND BUILD_MODULE)
     add_subdirectory(module)
