diff -ruN elfio-src/CMakeLists.txt elfio-modified/CMakeLists.txt
--- elfio-src/CMakeLists.txt    2025-08-20 14:09:18.620565726 -0700
+++ elfio-modified/CMakeLists.txt       2025-08-20 14:57:17.046009293 -0700
@@ -41,7 +41,7 @@

 target_include_directories(
     elfio
-    INTERFACE
+    SYSTEM INTERFACE
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
     $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
